File: Problem_set_1/Introductory_problems/p03_Repetition/Repetitions.java
package p03_Repetition;
import java.util.Scanner;

public class Repetitions{
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String input = scan.nextLine();

        int max = 1, current = 1;

        for(int i = 1; i < input.length(); i++){
            if(input.charAt(i) == input.charAt(i - 1)){
                current++;
            } else{
                max = Math.max(max, current);
                current = 1;
            }
        }

        if(current > max){
            max = current;
        }
        System.out.println(max);
        scan.close();
    }
}

File: Problem_set_1/Introductory_problems/p03_Repetition/Repetitions.py
str_input = str(input())

max_val = 1
current_val = 1

for ch in range(1, len(str_input)):
    if(str_input[ch] == str_input[ch - 1]):
        current_val += 1
    else:
        max_val = max(max_val, current_val)
        current_val = 1

if(current_val > max_val):
    max_val = current_val

print(max_val)